tests/particle_output_multiple_formats/statistics

```diff
@@ -0,0 +1,16 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Number of mesh cells
+# 4: Number of Stokes degrees of freedom
+# 5: Number of temperature degrees of freedom
+# 6: Number of degrees of freedom for all compositions
+# 7: Iterations for temperature solver
+# 8: Iterations for composition solver 1
+# 9: Iterations for Stokes solver
+# 10: Velocity iterations in Stokes preconditioner
+# 11: Schur complement iterations in Stokes preconditioner
+# 12: Time step size (seconds)
+# 13: Number of advected particles
+# 14: Particle file name(s)
```

Comment:
Currently, under the column name 'Particle file name(s)', the path to particle files are recorded in the format: `output-particle_output_multiple_formats/particles/particles-00000.{txt,h5,vtu}`. 
Is this good enough? 
I personally, dislike the length of the string and support the idea of creating a column header named, 'Particle directory <name>' and list `particles/particles-00000.{txt,h5,vtu}`. Which string do you guys prefer (@gassmoeller, @bangerth, @egpuckett)??